scripts/bump.sh

```diff
@@ -22,22 +22,26 @@ SHA=$(npx -q -p @prisma/cli@"$PRISMA_CHANNEL" prisma --version | grep "Query Eng
 NEXT_EXTENSION_VERSION=$3
 echo "NEXT_EXTENSION_VERSION: $NEXT_EXTENSION_VERSION"
 
-if [ "$CHANNEL" = "dev" ]; then
+if [ "$CHANNEL" = "insider" ]; then
     echo "$PRISMA_VERSION" > scripts/prisma_version_unstable
 else
     echo "$PRISMA_VERSION" > scripts/prisma_version_stable
 fi
 
-# If the channel is dev, we need to change the name, displayName to the dev extension
-if [ "$CHANNEL" = "dev" ]; then
+# If the channel is insider, we need to change the name, displayName to the insider extension
+if [ "$CHANNEL" = "insider" ]; then
     jq ".version = \"$NEXT_EXTENSION_VERSION\" | \
-        .name = \"prisma-dev\" | \
-        .displayName = \"Prisma Dev\"" \
+        .name = \"prisma-insider\" | \
+        .displayName = \"Prisma - Insider\" | \
+        .description = \"This is the Insiders Build.\" | \
```

Comment:
Can we elaborate a bit what this means?

Reply:
when an unstable version is published, the name, displayname and description gets changed in the package.json. This is the same procedure as before with `dev`.

Reply:
Ha sorry, not what I meant: Can we elaborate in the description what "Insider Build" means ;)


```suggestion
        .description = \"This is the Insider Build.\" | \
```

Reply:
Ah ok 😄 yes, the intention was to make it very clear that the Insider version is not the one users should normally be using.